static/js/zulip.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env commonjs */
+
+"use strict";
+
+const {Filter} = require("./filter");
+const message_list = require("./message_list");
+
+// This file is being eliminated as part of the general purge of
+// global variables from Zulip (everything is being moved into
+// modules).  Please don't add things here.
+
+const home_msg_list = new message_list.MessageList({
+    table_name: "zhome",
+    filter: new Filter([{operator: "in", operand: "home"}]),
+    excludes_muted_topics: true,
+    excludes_muted_users: true,
+});
+
+window.home_msg_list = home_msg_list;
+window.current_msg_list = home_msg_list;
```

Comment:
This looks like a failed merge conflict resolution; the code moved to `static/js/message_lists.js`.

Reply:
Yeah, my bad. Fixed.